build/Build.cs

```diff
@@ -102,7 +103,7 @@ ITargetDefinition CompileDefinition(ITargetDefinition targetDefinition, [CanBeNu
     Target TestWindows => _ => TestDefinition(_, Compile, null, runDotMemoryTests: true);
 
     [PublicAPI]
-    Target TestLinux => _ => TestDefinition(_, Compile, null, runDotMemoryTests: false);
+    Target TestLinux => _ => TestDefinition(_, Compile, "net6.0", runDotMemoryTests: false);
```

Comment:
Why does only TestLinux pass "net6.0" but TestWindows does not? Is a comment required?

Reply:
Ooh, that's a mistake. Nice catch.

Reply:
So the context here is that in the builds we actually call the specific targets for the various Windows tests `TestWindowsNet48` and `TestWindowsNet60`. Linux didn't have that distinction. The original goal here was to maintain the existing behaviour (that the Linux .NET6 tests only test that framework).

Still, this shouldn't be part of the SDK change.